quotes.json

```diff
@@ -0,0 +1 @@
+{"0":[{"quote":"Is"},{"quote":"A"},{"quote":"Gay"}],"1":[],"2":[],"3":[],"4":[]}
```

Comment:
quotes.json should probably be in .gitignore.

Reply:
Done